README.md

```diff
@@ -1,24 +1,249 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
```

Comment:
### I'm just using the readme file to write comments because not all files are included in this PR you should modify the code that is responsible for what is shown below 

- One of the requirements is to follow the given design by implementing the same colors, font-family, spaces, and layout you can see the screenshot of the requirements below 
<img width="546" alt="Screenshot 2023-02-15 at 15 24 42" src="https://user-images.githubusercontent.com/61790587/219056526-1c6f88ad-759e-45a9-805d-f8aa0f2e4ded.png">

- [ ] it would be better to stick to the design make the categories  page the same as in the design by adding the same font size, and spaces, and layout and making sure the add button should be the same as the one in the design 
- you should use the same colors also 

- Categories page design 
<img width="233" alt="Screenshot 2023-02-15 at 15 40 46" src="https://user-images.githubusercontent.com/61790587/219059522-8c6034e0-b221-4605-b2a1-ac510660b7b3.png">

- Yours 
![Screenshot 2023-02-15 at 15 40 40](https://user-images.githubusercontent.com/61790587/219059560-031ec9f6-65ea-487b-840f-3f60348d7af6.png)

- Font family from the Design 
<img width="412" alt="Screenshot 2023-02-15 at 15 25 53" src="https://user-images.githubusercontent.com/61790587/219057802-99df8d3a-3e32-4603-badf-f2e71e8bd4c0.png">

Reply:
- [ ] @NatanimA great work implementing the changes requested however the design needs some changes to match the design given,  below is a side-by-side comparison for better understanding comparison,


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/219135289-acfb40a3-b9e7-4a13-bb96-6a8103e8ac9b.png)  |  ![image](https://user-images.githubusercontent.com/61408860/219135346-b5717ef3-5357-48db-bbbf-bd5d2d7b6da5.png)